.circleci/config.yml

```diff
@@ -14,6 +14,7 @@ commands:
   test:
     steps:
       - run: npm run test:all
+      - run: npm run test:redis --prefix integrationTests
```

Comment:
I originally thought this was starting redis. Please consider a different name. Additionally, these should be run as a part of test:all ... otherwise it's not running all tests.

Reply:
redis is not started here, we are only testing redis

also we need to test redis separately as it needs docker

Reply:
Docker always starts as a part of starting circleci, so it can fall in to test:all.

Reply:
So following the format of the rest of the commands wouldn't we want test:all to run everything with the expectation that redis is running, and test:all:redis to start redis for local development?

Reply:
Thats were it gets complicated.. test:all calls integration:test:bundle that will test only basic files.. Let me c if it makes sense to refactor the scripts

Reply:
Can u check now, fixed some issues with the scripts and test:all now tests redis and test:all:redis starts and tests